docs/versioned_docs/version-v0.3/quickstart.mdx

```diff
@@ -14,7 +14,7 @@ Please be aware that this guide is not a replacement for studying the docs. You
 must understand core concepts and APIs to use ORY Kratos productively. This is
 merely a guide to get you set up with some examples.
 
-<iframe width="560" height="315" src="https://www.youtube.com/embed/5t1Zr_zJc7E" frameborder="0" allowfullscreen></iframe>
+<iframe width="560" height="315" src="https://www.youtube.com/embed/JhlIZ3aChXc" frameborder="0" allowfullscreen></iframe>
```

Comment:
But the video is not done with those versions, correct?

Reply:
Yes.

I can remove it from the older versions if that would be better

I could leave the old video in there as well, as it is only unlisted at the moment. 

But I would prefer to remove the old video, as it has the old logo and is just worse in general.

Reply:
Removed it from v0.3. 
v0.4 should be ok right?

Reply:
You made the video ;)